scripts/nn/util.dml

```diff
@@ -200,3 +200,19 @@ unpad_image = function(matrix[double] img_padded, int Hin, int Win, int padh, in
   }
 }
 
+threshold = function(matrix[double] X, double thresh)
+    return (matrix[double] out) {
+  /*
+   * Computes an indicator matrix with values in (0, 1) depending on whether or not the values in X
```

Comment:
Just for consistency with the rest of the library can you limit the line length of these docs to 72 characters?